iree/test/e2e/xla_ops/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ iree_check_single_backend_test_suite(
     "add.mlir"
     "broadcast.mlir"
     "broadcast_in_dim.mlir"
+    "compare.mlir"
```

Comment:
Doesn't it compare to master branch? I think this is not related to the change..

Reply:
That's from running `bazel_to_cmake.py`, which has had this diff since https://github.com/google/iree/commit/43b594747d3fc5593076f7120feebf4e0954be42. We could make that change separately, but it was easier to just include it while I'm already adjusting the tests in this area :P